defi/src/rwa/db.ts

```
@@ -241,19 +242,69 @@ export async function storeHistoricalPG(inserts: any, timestamp: number): Promis
+    if (process.env.RWA_REFILL) {
+        const insertIds = inserts.map((i: any) => i.id);
+        const existing = await DAILY_RWA_DATA.findAll({
+            attributes: ['id', 'mcap', 'activemcap', 'defiactivetvl'],
+            where: { timestamp: dayTimestamp, id: insertIds },
+            raw: true,
+        }) as any[];
+        existing.forEach((r: any) => { existingRecords[r.id] = r; });
+    }
+
     const dailyInserts: any[] = [];
     inserts.map((i: any) => {
```

Biome (2.4.6)

[error] This callback passed to map() iterable method should always return a value.

(lint/suspicious/useIterableCallbackReturn)